docs/README.md

```diff
@@ -86,13 +96,24 @@ node {
 
 ### Username with Password
 
-A username and password pair.
+A *username* and *password* pair.
+
+Value: *password*
+
+Tags:
+
+- `jenkins:credentials:type` = `usernamePassword`
+- `jenkins:credentials:username` = *username*
+
+#### Example
+
+AWS CLI:
 
 ```bash
-aws secretsmanager create-secret --name 'artifactory' --secret-string 'supersecret' --tags 'Key=jenkins:credentials:username,Value=joe' --description 'Acme Corp Artifactory login'
+aws secretsmanager create-secret --name 'artifactory' --secret-string 'supersecret' --tags 'Key=jenkins:credentials:type,Value=usernamePassword,Key=jenkins:credentials:username,Value=joe' --description 'Acme Corp Artifactory login'
```

Comment:
So testing out a little more and I found that you need to space separate each of the tags for all of them to apply.  I would give this a quick test on your machine to make sure it does what you expect.
```suggestion
aws secretsmanager create-secret --name 'artifactory' --secret-string 'supersecret' --tags 'Key=jenkins:credentials:type,Value=usernamePassword' 'Key=jenkins:credentials:username,Value=joe' --description 'Acme Corp Artifactory login'
```